src/App.js

```diff
@@ -234,6 +246,23 @@ export const App = () => {
 					"verb": operation,
 				}
 			}]
+		} else if (type === 'object' && params.objectId) {
+			body = [{
+				"object": [{
+					"operation": 'GET',
+					"action": "ALLOW",
+					"filters": [{
+						"headerType": "OBJECT",
+						"key": "$Object:objectID",
+						"matchType": "STRING_EQUAL",
+						"value": params.objectId,
```

Comment:
I noticed that access filter rules do not affect a specific object and give access to all objects in all containers with this token, what am I doing wrong here?

Reply:
You need an additional rule to forbid access for OTHERS right after the "good" one.